### PR TITLE
[mobile] Update button, radio, checkbox in dialog sample

### PR DESCRIPTION
**Pull request by DohyungLim**
- 3 Stacked button implementation.
- OneUI button, radio, checkbox applied to all samples.
- List spacing adjustment.
- Delete unnecessary code.

### CODE REVIEW

File: src/css/profile/mobile/common/popup.less [hunk 1]
```diff
@@ -91,24 +91,23 @@
...
-			height: 32 * @px_base;
-		}
-		input[type=radio] {
-			margin: 0 12 * @px_base 0 0;
-			vertical-align: -10 * @px_base;
-			width: 32 * @px_base;
-			height: 32 * @px_base;
+
+		.ui-listview.ui-content-area {
+			margin-bottom: 0 * @px_base;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: You can set the value with '0'.

File: src/css/profile/mobile/common/footer.less [hunk 2]
```diff
@@ -1,17 +1,16 @@
 .LESSui-footer {
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/footer.less]: `.LESSui-footer` is for the 'ui-footer'.
Is it need to change the value here?

File: src/css/profile/mobile/common/popup.less [hunk 3]
```diff
@@ -129,6 +128,23 @@
...
+			background-color: var(--on-off-switch-divider-color);
+			opacity: var(--on-off-switch-divider-opacity);
+		}
+
+		.ui-popup-stack .ui-btn {
+			margin-bottom: 16 * @px_base;
+		}
+		.ui-popup-stack .ui-btn:nth-last-child(1) {
+			margin-bottom: 0;
+		}
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less:179]: You can reduce code with
```
	.ui-popup-stack .ui-btn {
		margin-bottom: 16 * @px_base;

		&:nth-last-child(1) {
			margin-bottom: 0;
		}
	}
```

File: src/css/profile/mobile/common/popup.less [hunk 4]
```diff
@@ -129,6 +128,23 @@
 		background-color: @color_popup_footer_bg;
 		position: sticky;
 		.LESSui-footer();
+
+		div.ui-li-divider::after {
+			content: "";
+			position: absolute;
+			width: 1 * @px_base;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: You can set the value with '@px_base' only.

File: src/css/profile/mobile/common/popup.less (hunk 1 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

File: src/css/profile/mobile/common/popup.less (hunk 3 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less:179]: done.

File: src/css/profile/mobile/common/popup.less (hunk 4 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: @HunseopJeong 
I know. Is there a problem?

File: src/css/profile/mobile/common/footer.less (hunk 2 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/footer.less]: To apply OneUI to the buttons used in the footer, modification is required.

File: examples/mobile/UIComponents/components/dialogs/dialog.html [hunk 5]
```diff
@@ -93,13 +93,10 @@ <h1>
...
 				</div>
-				<div class="ui-popup-footer ui-grid-col-2">
-					<a class="ui-btn" data-rel="back">
-						Action1
-					</a>
-					<a class="ui-btn" data-rel="back">
-						Action2
-					</a>
+				<div class="ui-popup-footer">
+					<button class="ui-btn" data-style="flat">Action 1</button>
```
Comment (singa2000) [inline examples/mobile/UIComponents/components/dialogs/dialog.html:97]: How about to remain the date-rel="back" ?
date-rel="back" means , if push the button , the popup is closed?

File: examples/mobile/UIComponents/components/dialogs/dialog.html [hunk 6]
```diff
@@ -348,16 +334,35 @@ <h1>
...
 				</div>
 				<div class="ui-popup-content">
-					<div class="ui-popup-controlbox">
-						<div><input type="radio" name="test"/>List</div>
-						<div><input type="radio" name="test"/>Detailed list</div>
-						<div><input type="radio" name="test"/>Grid</div>
-					</div>
+					<ul class="ui-listview ui-content-area">
+						<li class="ui-li-has-radio">
+							<input checked="checked" id="radio-1" name="radio-group" type="radio" />
```
Comment (singa2000) [inline examples/mobile/UIComponents/components/dialogs/dialog.html]: By default, it look "List" has the "checked" value.
But in the test, it don't display the default check.
Would you check it, please?

File: src/css/profile/mobile/common/popup.less (hunk 4 above)
Comment (HunseopJeong) [reply src/css/profile/mobile/common/popup.less]: You can use the "@px_base" only not the "1 * @px_base".

File: src/css/profile/mobile/common/footer.less (hunk 2 above)
Comment (HunseopJeong) [reply src/css/profile/mobile/common/footer.less]: I think .LESSui-footer is used in other components.
Need to check it affect to other components.

Comment (DohyungLim) [reply src/css/profile/mobile/common/footer.less]: I'm restoring the code to not modify the .LESSui-footer so it doesn't create a problem.
Change ui-popup-footer to control the properties of .ui-btn directly.

File: src/css/profile/mobile/common/popup.less (hunk 4 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

File: examples/mobile/UIComponents/components/dialogs/dialog.html (hunk 5 above)
Comment (DohyungLim) [reply examples/mobile/UIComponents/components/dialogs/dialog.html:97]: date-rel = "back" works on tag a. It doesn't work in button tag.

File: examples/mobile/UIComponents/components/dialogs/dialog.html (hunk 6 above)
Comment (DohyungLim) [reply examples/mobile/UIComponents/components/dialogs/dialog.html]: checked.

done.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**